lazyflow/operators/opPatchCreator.py

```diff
@@ -196,8 +197,8 @@ def execute(self, slot, subindex, roi, result):
             gWidth = self.GridWidth.value
             gHeight = self.GridHeight.value
 
-            numPatchesVertical = int((gHeight - pHeight) / skipVertical) + 1
-            numPatchesHorizontal = int((gWidth - pWidth) / skipHorizontal) + 1
+            numPatchesVertical = (gHeight - pHeight) // skipVertical) + 1
```

Comment:
I commented because I did not see a related test failure, this might in fact be untested...